### PR TITLE
feat: admin left navbar component

### PR DESCRIPTION
**Pull request by saidabdul80**

### CODE REVIEW

File: app/Http/Controllers/Api/AuthController.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function uuid()
     public function login(Request $request)
     {
         $validator = Validator::make($request->all(), [
-            'username' => 'required',
+            'email' => 'required',
```
Comment (Auwalms) [inline app/Http/Controllers/Api/AuthController.php]: @saidabdul80 we are using username in this case cos students will use Matric Number while staffs can use their email as their username. Please revert this line.

File: app/Http/Controllers/Api/AuthController.php [hunk 2]
```diff
@@ -44,7 +44,7 @@ public function login(Request $request)
         }
 
         $credentials = [
-            'username' => $request->get('username'),
+            'email' => $request->get('email'),
```
Comment (Auwalms) [inline app/Http/Controllers/Api/AuthController.php]: Same applies to this please @saidabdul80

File: app/Http/Controllers/Api/CourseController.php [hunk 3]
```diff
@@ -20,15 +20,14 @@ class CourseController extends Controller
...
-        //return dd($request->all());
         $validator = Validator::make($request->all(), [                        
             'title' => 'required',
             'code' => 'required',
             'resource_url' => 'required',
             'experiment_id' => 'required',
             'instructor_id' => 'required',
         ]);
-        $school_id = School::first();        
+        $school_id = School::first()->id;
```
Comment (Auwalms) [inline app/Http/Controllers/Api/CourseController.php:30]: Thanks for this.

File: app/Http/Controllers/Api/CourseController.php [hunk 4]
```diff
@@ -40,12 +39,11 @@ public function create(Request $request)
         $code = $request->get('code');
         $description = $request->get('description');
         $status = $request->get('status') ?? 'Active';
-
         
-        $checkCourse = Course::where(['code' => $code])->first()->id;
+        //return response()->json(['error' => Course::where(['code'=>$code])->first()], 200);
+        $checkCourse = Course::where(['code'=>$code])->first();
```
Comment (Auwalms) [inline app/Http/Controllers/Api/CourseController.php:44]: Glad you checked and corrected this.

**Review by Auwalms — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Api/AuthController.php (hunk 1 above)
Comment (saidabdul80) [reply app/Http/Controllers/Api/AuthController.php]: I thought backend has been separated, I just used that as test in other not to change much things. However I will try to do git ignore to all backend files

File: app/Http/Controllers/Api/AuthController.php (hunk 2 above)
Comment (saidabdul80) [reply app/Http/Controllers/Api/AuthController.php]: Noted

File: app/Http/Controllers/Api/CourseController.php (hunk 3 above)
Comment (saidabdul80) [reply app/Http/Controllers/Api/CourseController.php:30]: 🤗

File: app/Http/Controllers/Api/CourseController.php (hunk 4 above)
Comment (saidabdul80) [reply app/Http/Controllers/Api/CourseController.php:44]: Ok sir

**Review by Auwalms — APPROVED** [review]
LGTM

**Review by abdullahikawu — APPROVED** [review]
Looks good.

**Pull request merged**